stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Channels.kt

```diff
@@ -559,11 +557,6 @@ class Channels(val client: ChatClient, val channelClient: ChannelClient) {
                         val mutes: List<ChannelMute> = user.channelMutes
                     }
                 }
-
-            // Get updates about muted channels
-            client.subscribeFor<ChannelsMuteEvent> { event ->
```

Comment:
Same here

Reply:
Updated to `NotificationChannelMutesUpdatedEvent` event.